File: cSharp/problem-1002.cs
using System; 

// https://www.beecrowd.com.br/judge/en/problems/view/1002
class URI {

    static void Main(string[] args) { 

        double radius = double.Parse(Console.ReadLine()),
        pi = 3.14159,
        circumferenceArea = pi * Math.Pow(radius, 2);

        Console.WriteLine($"A={circumferenceArea.ToString("0.0000")}");

    }

}

File: c/problem-1002.c
#include <stdio.h>

// https://www.beecrowd.com.br/judge/en/problems/view/1002

int main() {
    double radius, circumferenceArea, PI = 3.14159;
    scanf("%lf", &radius);
    circumferenceArea = PI * (radius * radius);
    printf("A=%.4lf\n",circumferenceArea);

    return 0;
}